File: python/fizzBuzz.py
def fizzBuzz(limit):
  output = "";
  for n in range (0, limit):
    if n % 3 == 0 :output += "Fizz"
    if n % 5 == 0: output += "Buzz"
    if not (n % 3 == 0 or n % 5 == 0): output += str(n)
    output += '\n'
  return output

def commonFizzBuzz(limit):
  n = 0
  output = ""
  while n < limit:
    if n % 5 == 0 and n % 3 == 0: output += 'FizzBuzz'
    elif n % 5 == 0: output += 'Buz'
    elif n % 3 == 0: output += 'Fiz'
    else: output +=str(n)
    output += '\n'
    n+=1
  return output

print(fizzBuzz(20))

File: java/fizzBuzz.java
class Excercises{

  public static String fizzBuzz(int limit) {
    String output = "";
    for (int n = 0; n <= limit; ++n) {
      if (n % 3 == 0) output += "Fizz";
      if (n % 5 == 0) output += "Buzz";
      if (!(n % 3 == 0 || n % 5 == 0)) output +=  n;
      output += "\n";
    }
    return output;
  }

  public static String commonFizzBuzz(int limit) {
    int n = 0;
    String output = "";
    while (n < limit) {
      if (n % 5 == 0 && n % 3 == 0) output += "FizzBuzz";
      else if (n % 5 == 0) output += "Buzz";
      else if (n % 3 == 0) output += "Fizz";
      else output += n;
      output += "\n";
      ++n;
    }
    return output;
  }
  public static void main(String [] argv){
    System.out.println(fizzBuzz(15));
  }
}



// console.log(commonFizzBuzz(20));
